index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Awesome Book List</title>
+  </head>
+  <body>
+    <main>
+      <navbar class="navbar">
+        <h3 class="nav-title">Awesome Books</h3>
+        <div class="nav-items">
+          <a href="#book-list"><div class="nav-list">List</div></a>
+          <a href="#book-form"><div class="nav-add">Add New</div></a>
+          <a href="#contact"><div class="nav-contact">Contact</div></a>
+        </div>
+      </navbar>
+      <p id="date-time"></p>
+      <section class="main-container">
+        <div class="form-container">
+          <h1>Add a new book</h1>
+          <form id="book-form">
+            <input type="text" id="title" placeholder="Title" />
+            <input type="text" id="author" placeholder="Author" />
+            <button type="submit"><p>Add</p></button>
```

Comment:
This button shouldn't be centered, kindly move it to the right just as we have it in the wireframe below;

Wireframe:
<img width="405" alt="image" src="https://user-images.githubusercontent.com/31897434/217213086-16783152-94a8-449a-a657-2495c0e67513.png">


Your design:
<img width="590" alt="image" src="https://user-images.githubusercontent.com/31897434/217213156-b5b0ef74-dacc-4038-b0e9-e5d609ef5d7a.png">